dev-python/capturer/capturer-2.4.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Easily capture stdout/stderr of the current process and subprocesses"
+HOMEPAGE="https://capturer.readthedocs.io/en/latest/
+	https://pypi.org/project/capturer/
+	https://github.com/xolox/python-capturer"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+IUSE="test"
```

Comment:
This is set up by `distutils_enable_tests`-

Reply:
Yes, but I need to have `IUSE="test"` set before `test?` in `DEPEND`.

I either have to put `distutils_enable_tests` before `DEPEND` in which case it will append IUSE=test before the call to `test?`, but the packages that `distutils_enable_tests` appends to `DEPEND` would be overwritten in this case.

Or I have `distutils_enable_tests` after `DEPEND`, in which case the packages that are appended to `DEPEND` are not overwritten, but then I have to manually specify IUSE=test to use `test?`.

I have chosen for the latter option here, but I could change it to the former. However, this is in the comments in the distutils-r1 eclass:
```
# This function must be called in global scope, after RDEPEND has been
# declared.  Take care not to overwrite the variables set by it.
```

Reply:
It **shouldn't** be a problem, but granted I haven't tested it. The ebuild is sourced before anything is run, so it should see defined variables.

Reply:
Hmm, intresting, it seems to work just fine without `IUSE=test`. At least repoman/emerge aren't complaining, I'm not sure where I got the idea from that it doesn't work.